include/rserver.hh

```diff
@@ -39,25 +43,31 @@ class RServerRPC final : public PLContainer::Service {
 
    private:
     PlcRuntime *runtime;
-    RServerLog rLog;
+    RServerLog *rLog;
 };
 
 class RServer {
    public:
-    explicit RServer(bool standAloneMode) { this->standAloneMode = standAloneMode; }
+    RServer(bool standAloneMode, RServerLog *rLog) {
+        this->standAloneMode = standAloneMode;
+        this->rLog = rLog;
+        server = new RServerRPC(rLog);
```

Comment:
should delete the point in deconstruct function if this point is allocated in constructor.

Reply:
Thanks fixed.